src/quo2/components/inputs/address_input/view.cljs

```diff
@@ -0,0 +1,161 @@
+(ns quo2.components.inputs.address-input.view
+  (:require [react-native.core :as rn]
+            [react-native.clipboard :as clipboard]
+            [quo2.theme :as quo.theme]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.icon :as icon]
+            [quo2.components.buttons.button.view :as button]
+            [quo2.components.inputs.address-input.style :as style]
+            [utils.i18n :as i18n]
+            [reagent.core :as reagent]))
+
+(def ^:private ens-regex #"^(?=.{5,255}$)([a-zA-Z0-9-]+\.)*[a-zA-Z0-9-]+\.[a-zA-Z]{2,}$")
```

Comment:
should this be defined somewhere more common? 🤔

Reply:
could also be passed in as a prop if we need different regex dependent on networks/ test networks 🤔